Changelog.md

```diff
@@ -13,6 +13,8 @@ New
   ([#437])
 * The RRDP client now supports the gzip transfer encoding for HTTPs.
   ([#463], contributed by [@bjpbakker])
+* The `exception` config file value now also accepts a single string with
+  a path name in addition to an array of strings. ([#471])
```

Comment:
"in addition to" makes it sound like you can pass something like `exception = "/some/path", [ "some", "array", "values" ]` which I guess is not the case. Perhaps "instead of" rather than "in addition to" ?